k8s/aat/common/monitoring/oneagent.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: helm.fluxcd.io/v1
```

Comment:
thoughts on moving this to kustomize? since we're at 3 environments now?

Reply:
Yeah, I guess you are right. I will move this to Kustomize.